src/Mapbender/PrintBundle/Component/PrintService.php

```diff
@@ -271,27 +271,29 @@ private function getImages($width, $height)
         foreach ($this->mapRequests as $i => $url) {
             $this->getLogger()->debug("Print Request Nr.: " . $i . ' ' . $url);
 
-            $mapRequestResponse = $this->mapRequest($url);
+            try {
+
+                $mapRequestResponse = $this->mapRequest($url);
+            } catch (\Exception $e) {
+                continue;
+            }
 
             $imageName    = $this->makeTempFile('mb_print');
-            $imageNames[] = $imageName;
             $rawImage = $this->serviceResponseToGdImage($imageName, $mapRequestResponse);
```

Comment:
You disabled temp file cleanup for images that downloaded but could not be opened by GD. Why?